systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -118,6 +119,7 @@ function checkDirty(obj)
 
     function model = compile(model)
       w = warning('off','Drake:RigidBodyManipulator:UnsupportedContactPoints');
+      model.gurobi_present = checkDependency('gurobi');
```

Comment:
With this commit, we get the `l_init_called failed` error on the Atlas computers every time we try to start up the controller (specifically, every time we try to enter `constructQPDataPointerMex`. Switching it to `model.gurobi_present = true` prevents that error from occurring.